### PR TITLE
Fix bad labview link

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

**Comment by sciencewhiz** [comment]
I think there should be a todo for when @Kevin-OConnor gets a response for where the new location is rather then completely deleting it.

**Comment by Daltz333** [comment]
Done

File: source/docs/software/labview/resources/labview-resources.rst [hunk 1]
```diff
@@ -10,7 +10,9 @@ LabVIEW Resources
 LabVIEW Basics
 --------------
 
-NI provides a `combination of videos <https://www.ni.com/academic/students/learn-labview/>`_ and `traditional text/picture tutorials on the basics of LabVIEW <https://www.ni.com/getting-started/labview-basics/>`_. These tutorials can help you get acquainted with the LabVIEW environment and the basics of the graphical, dataflow programing model used in LabVIEW.
+NI provides `tutorials on the basics of LabVIEW <https://www.ni.com/getting-started/labview-basics/>`_. These tutorials can help you get acquainted with the LabVIEW environment and the basics of the graphical, dataflow programing model used in LabVIEW.
+
+.. todo:: Use student videos if kevin gets the link fixed
```
Comment (sciencewhiz) [inline source/docs/software/labview/resources/labview-resources.rst:15]:
```suggestion
.. todo:: Add academic videos when link fixed https://www.ni.com/academic/students/learn-labview/
```

Comment (Daltz333) [reply source/docs/software/labview/resources/labview-resources.rst:15]: Linkcheck will check todo links

Comment (sciencewhiz) [reply source/docs/software/labview/resources/labview-resources.rst:15]: even if it's not formatted as a link?

Comment (Daltz333) [reply source/docs/software/labview/resources/labview-resources.rst:15]: Correct.

Comment (sciencewhiz) [reply source/docs/software/labview/resources/labview-resources.rst:15]: Edited my suggestion and confirmed it passed linkcheckdiff

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**